Packs/CortexXDR/ReleaseNotes/5_2_4.md

```diff
@@ -0,0 +1,41 @@
+
+#### Mappers
+##### XDR - Incoming Mapper
+Added support for the **Cortex XDR - Lite** incident type.
+
+#### Integrations
+##### Palo Alto Networks Cortex XDR - Investigation and Response
+- Updated the default value for the classifier and incident type to `Cortex XDR - Lite`.
+- Updated the Docker image to: *demisto/python3:3.10.13.78960*.
+
+
+#### Scripts
+##### New: InvestigationVerdict
+New: This widget displays the Verdict.
```

Comment:
add the layout you are using for the widget

Reply:
done